### PR TITLE
Bump dependencies in GitHub Actions with dependabot

### LINKED ISSUE #419: Bump dependencies in GitHub Actions with dependabot
This can be done by adding this in `.github/dependabot.yml`'s `updates:` list:

```yml
- package-ecosystem: github-actions
  directory: "/"
  schedule:
    interval: daily
  labels:
    - "area: dependencies"
```

Because PRs like #416 shouldn't be made by hand.

### PR DESCRIPTION
**Pull request by PerchunPak**
Fixes #419.

### CODE REVIEW

**Review by ItsDrike — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -11,3 +11,9 @@ updates:
       interval: "daily"
     labels:
       - "area: dependencies"
+  - package-ecosystem: github-actions
```
Comment (ItsDrike) [inline .github/dependabot.yml:14]: Actually I missed this issue initially, seems like this needs to be a string.
```suggestion
  - package-ecosystem: "github-actions"
```

**Review by ItsDrike — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (kevinkjt2000) [reply .github/dependabot.yml:14]: In yaml, everything is a string by default?

Comment (ItsDrike) [reply .github/dependabot.yml:14]: Ah, I was going off of [these docs](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot) which had the string quoted. Alright then, although for consistency, it should probably still be quoted, just like in the above `package-ecosystem: "pip"`

Comment (PerchunPak) [reply .github/dependabot.yml]: Done [here](https://github.com/py-mine/mcstatus/compare/4e6e7a426d219d2a70e760484794b5c32bdb82c4..8b58ed95745369197d14f330ecd3ee7899b87157).

**Review by ItsDrike — APPROVED** [review]

**Pull request merged**